Dockerfile.dapper

```diff
@@ -27,8 +31,7 @@ ARG DAPPER_HOST_ARCH
 ARG DOWNLOADS_DIR=/usr/src/downloads
 ARG KERNEL_ARCH=arm64
 ARG KERNEL_URL=https://github.com/raspberrypi/linux.git
-ARG KERNEL_BRANCH=rpi-4.14.y
-ARG KERNEL_VERSION=4.14.114
+ARG KERNEL_BRANCH=rpi-4.19.y
```

Comment:
Please do not upgrade the kernel version although this is a meaningful thing, because `rancher/os` expects to upgrade the kernel after a series of stability tests.

Reply:
Version bump was in service to getting things working on the Pi4, but that's probably out of scope for this PR anyway. Is leaving the minor version floating and just pinning the branch okay?